### ISSUE TITLE
Unplayable Lag When Viewing Enemy Ogre Magi + Invoker Lotus Bug

### ISSUE THREAD

**Issue by kiwixdolceh** [ValveSoftware/Dota2-Gameplay#14394]
### Description

I am the Invoker player in this game. What was otherwise a fine game at first was made unplayable due to a bug or cheat with the enemy Ogre Magi (and less frequently with the enemy Faceless Void, still not completely sure). When viewing these heroes (perhaps due to their Arcanas), my game framerate plummeted to <10 fps, as did the framerate of my friend playing with me (Earthshaker). When not viewing these heroes, the game ran as normal. But even when they were visible and then became hidden by fog of war, my game frame rate would suddenly recover as soon as Ogre Magi or Faceless Void were off-screen. I had no similar issues in my other two games just prior to this one.

In addition to this, also encountered a bug where I could not pick up Lotus from Dire-side Lotus Pool (as Invoker) while my friend (as Pudge) could. I assume this is an unrelated bug but I encountered it in the same game.

Lastly, got muted mid-game after voicing my frustrations about the bugs in chat, but I assume this is a temporary thing with the auto-moderation due to my otherwise above 10,000 behavior and communication score. Unsure, but only added to the frustrating experience.

I have also submitted issue reports with Valve and Steam.

### Example Match ID (and possibly Timestamp)

7493908645

**Comment by kiwixdolceh** [comment]
<img width="1308" alt="Screenshot 2023-12-17 at 11 52 36 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/154115815/5882d505-dbba-4eb8-ba33-46201c085e99">

**Comment by BackToLosing** [comment]
Did they have the Ogre Immortal Gemlick Decanter. I thought it was the Arcana and after further testing it was the Immortal.

**Comment by kiwixdolceh** [comment]
> Did they have the Ogre Immortal Gemlick Decanter. I thought it was the Arcana and after further testing it was the Immortal.

Yes, they also had the Gemlick Decanter. Also the Auspice of the Whyrlegyge immortal. I’m assuming I *thought* faceless void was an issue since void lanes with ogre and stuck with him most the game so I probably couldn’t differentiate the problem source. Thanks for the response so maybe I wasn’t just going crazy lol.
